themes-default/slim/src/app.js

```diff
@@ -69,6 +70,9 @@ const app = new Vue({
     name: 'App',
     store,
     router,
+    components: {
+        Truncate
```

Comment:
There should be an option to import the component as a plugin (look at the source code).
So it wouldn't use the `components` property, but rather use `Vue.use` like `toggle-button`. It's much cleaner IMO.

Reply:
It'd make more sense to use components than register it globally. I'd like to remove all global registrations eventually.

Reply:
I agree about the global components, since you'd (probably) only use it in the `showheader` component (exactly like you do here) once it has been Vueified.
My point is, for now I think it's much cleaner to use `Vue.use(...)` for vendored components if possible.

Reply:
It's should produce exactly the same result BTW - `Vue.use` registers globally, just like putting the `components` property on the main Vue instance.